src/Components/Publishing/EditorialFeature/Components/Eoy2018Artists/index.tsx

```diff
@@ -100,6 +101,7 @@ export class Eoy2018Artists extends React.Component<EditorialFeaturesProps> {
           <Text
             html={section.body}
             layout={layout}
+            isMobile={isMobile}
```

Comment:
We will probably want to use this in the `isTablet` use case as well.

Reply:
Good call, in that case maybe I can just reuse showToolTips and get rid of this prop

Reply:
This seems addressed but lmk if I've missed something @eessex!